src/ConnectedMode/UI/Resources/UiResources.resx

```diff
@@ -180,4 +180,19 @@
   <data name="Username" xml:space="preserve">
     <value>Username</value>
   </data>
+  <data name="SelectOrganizationLabel" xml:space="preserve">
+    <value>Choose one of your organizations:</value>
+  </data>
+  <data name="OrganizationSelectionDialogTitle" xml:space="preserve">
+    <value>Select SonarCloud Organization</value>
+  </data>
+  <data name="OpenManualOrganizationSelectionButton" xml:space="preserve">
+    <value>Select another organization...</value>
```

Comment:
I wouldn't add "..." in the button name

Reply:
Copied that from intellij
![image](https://github.com/user-attachments/assets/6bd94fe1-03b6-4718-8f00-d5165ee26f02)